javascript/packages/orchestrator/src/network.ts

```diff
@@ -95,16 +95,23 @@ export class Network {
   wasRunning: boolean;
   tmpDir: string;
   backchannelUri: string = "";
+  chainId?: string;
   chainSpecFullPath?: string;
   tracing_collator_url?: string;
   networkStartTime?: number;
 
-  constructor(client: Client, namespace: string, tmpDir: string) {
+  constructor(
+    client: Client,
+    namespace: string,
+    tmpDir: string,
+    startTime: number = new Date().getTime(),
+  ) {
     this.client = client;
     this.namespace = namespace;
     this.launched = false;
     this.wasRunning = false;
     this.tmpDir = tmpDir;
+    this.networkStartTime = startTime;
```

Comment:
```suggestion
  constructor(
    private readonly client: Client,
    private readonly namespace: string,
    private readonly tmpDir: string,
    private readonly networkStartTime: number = new Date().getTime(),
  ) {
```

Reply:
In case we proceed with this, the `readonly` values need to be removed from the declaration of the class

Reply:
It's a question of preferent but imho, it removed the boring part of plugging all property in the property definition so we have 2x less code

Reply:
Could not agree more - I do not mind any of the 2 (and since ts -> CommonJS it does not really matter the output size); All I am saying is that we should be consistent across repo

Reply:
I'm agree that we should stick to one option, at the moment we weren't using `readonly` properties in class constructors. But I see the benefit to switch to use it. I think we can address that in a separate `pr` if you both are agree?

Reply:
Well, in fact I use readonly because most of the time my properties are shared non-mutable state but if you want to define a property mutable, you just remove the readonly (it's a mechanical habit kind of ahah), only putting a modifier like private/public/protected define the property automatically on the class, readonly is a nice-to-have to ensure no modification of non-mutable state